arwen/contexts/output.go

```diff
@@ -292,6 +297,13 @@ func (context *outputContext) TransferValueOnly(destination []byte, sender []byt
 	senderAcc.BalanceDelta = big.NewInt(0).Sub(senderAcc.BalanceDelta, value)
 	destAcc.BalanceDelta = big.NewInt(0).Add(destAcc.BalanceDelta, value)
 
+	context.WriteLogWithIdentifier(
+		context.host.Runtime().GetContextAddress(),
+		[][]byte{destination, sender, value.Bytes()},
```

Comment:
Isn't it more intuitive to have the sender, then the destination?

Reply:
updated